src/components/Dropdown/Dropdown.tsx

```diff
@@ -20,6 +20,7 @@ const Dropdown: React.FC<IDropdown> = ({
     icon,
     isDisabled = false,
     isLabelFixed = true,
+    isLabelVisible = true,
```

Comment:
could `isLabelVisible` maybe `hasLabel`?

and could that be set dynamically by `label` being an optional prop? That would be killer DX

`const hasLabel = Boolean(label)` for example

Reply:
Thank you @BillyG83 
If we do a `label` is not mandatory, we even don't need to have `isLabelVisible` and `hasLabel`.

But it is called `isLabelVisible` and not `hasLabel` because it can have a label but in specific conditions, we may want to hide the label.

Reply:
![image](https://user-images.githubusercontent.com/78314301/157534066-b9ed3591-abbb-4635-aa31-09096bcaa6fc.png)

Reply:
e.g. full width submenu and collapsed

Reply:
I think we don't really need `isLabelFixed`, because we can make it conditional @rayyan224 what do you think?

Reply:
Yes, of course we can just use two Dropdowns and two different arrays of options, but do you think `isLabelVisible` a bad solution?

Reply:
Hmm by conditionally rendering do you mean having the label withing the text for the option item ?